plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/params/StartContainerParams.java

```diff
@@ -0,0 +1,47 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.plugin.docker.client.params;
+
+import org.eclipse.che.plugin.docker.client.json.HostConfig;
+
+/**
+ * Arguments holder for {@link org.eclipse.che.plugin.docker.client.DockerConnector#startContainer(StartContainerParams)}.
```

Comment:
You can see [here](https://docs.docker.com/engine/reference/api/docker_remote_api_v1.22/#start-a-container) that HostConfig is supported for backward compatibility only.
I think you should not add HostConfig to StartContainer params because we don't need such support.